### PR TITLE
Add the 3.3.0 release note

### PR DESCRIPTION
**Pull request by praneesha**
Add the 3.3.0 release note

### CODE REVIEW

**Review by keizer619 — APPROVED** [review]

File: en/docs/release-notes/version-3.3.0.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+- A powerful graphical editor, which allows you to easily create complex data transformations in a few minutes
+- Provides descriptive diagnostics, error highlighting, and code actions, which leverage the editing experience
+- An integrated expression editor to add custom values with the support of lang server suggestions, library browsing, etc.
+- Currently, supports JSON to JSON transformations and will support more in the future
+
+## Integrated GraphQL Tryit
+![graphql-tryit](./../../resources/release-notes/3.3.0/graphql-tryit.gif)
+
+- You can try out the GraphQL services using the `Tryit` codelens
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:24]: Should be `CodeLens`

File: en/docs/release-notes/version-3.3.0.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+- You can try out the GraphQL services using the `Tryit` codelens 
+- You can send and test any request with headers to your GraphQL service in VS Code itself without using any third-party tools
+- The GraphQL explorer will help you to explore the available endpoints and generate the query for you
+
+## Record editor
+![record-editor](./../../resources/release-notes/3.3.0/record-editor.gif)
+
+- The expression editor is now combined with the record editor, which will give a better editing experience with suggestions and library support
+- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
+- Record creation using JSON samples now supports separate record creation for complex JSON records.
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:33]: Inconsistent punctuation. Some bullet points on this page are ended with a full stop, while others are not.

File: en/docs/release-notes/version-3.3.0.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+- The GraphQL explorer will help you to explore the available endpoints and generate the query for you
+
+## Record editor
+![record-editor](./../../resources/release-notes/3.3.0/record-editor.gif)
+
+- The expression editor is now combined with the record editor, which will give a better editing experience with suggestions and library support
+- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
+- Record creation using JSON samples now supports separate record creation for complex JSON records. 
+
+## Project Design view (experimental)
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:35]: Shouldn't it be `Project design view`?

File: en/docs/release-notes/version-3.3.0.md [hunk 4]
```diff
@@ -1 +1,42 @@
-# Version 3.3.0
\ No newline at end of file
+# Ballerina VSCode extension (version 3.3.0) 
+
+We are happy to announce the Ballerina VS Code plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
+
+- [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
+- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates trying out the GraphQL services with the integrated client 
+- [Improved record editor](#record-editor) - Provides a better editing experience with suggestions 
+- [Project Design View](#record-editor) - Facilitates visualizing service interactions in your project
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:8]: Should be `Project design view`

File: en/docs/release-notes/version-3.3.0.md [hunk 5]
```diff
@@ -1 +1,42 @@
-# Version 3.3.0
\ No newline at end of file
+# Ballerina VSCode extension (version 3.3.0) 
+
+We are happy to announce the Ballerina VS Code plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
+
+- [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:5]: Should this be `Visual data mapper`?

File: en/docs/release-notes/version-3.3.0.md [hunk 6]
```diff
@@ -1 +1,42 @@
...
+- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates trying out the GraphQL services with the integrated client
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:6]: Texts in the link and text in the corresponding titles are not matching.

File: en/docs/release-notes/version-3.3.0.md [hunk 7]
```diff
@@ -1 +1,42 @@
...
+- Record creation using JSON samples now supports separate record creation for complex JSON records. 
+
+## Project Design view (experimental)
+![design-view](./../../resources/release-notes/3.3.0/design-view.gif)
+
+- This release includes an experimental feature that allows you to visualize service interactions in your project
+- It also comes with a view to see record-type compositions and relationship
+- Use the "Ballerina: Project Design" command to open design view
+
+Apart from the above, there are lots of other bug fixes and improvements. For more details, see the [issue list](https://github.com/wso2/ballerina-plugin-vscode/issues?q=is%3Aissue+is%3Aclosed).
```
Comment (anupama-pathirage) [inline en/docs/release-notes/version-3.3.0.md:42]: `many` seems better than `lots of`

File: en/docs/release-notes/version-3.3.0.md (hunk 3 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:35]: As this is a part of the UI, can special-case it as a pronoun and say `Project Design View`.

File: en/docs/release-notes/version-3.3.0.md (hunk 4 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:8]: As this is a part of the UI, can special-case it as a pronoun and say Project Design View.

File: en/docs/release-notes/version-3.3.0.md (hunk 5 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:5]: As this is a part of the UI, can special-case it as a pronoun and say `Visual Data Mapper`.

File: en/docs/release-notes/version-3.3.0.md (hunk 1 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:24]: Fixed.

File: en/docs/release-notes/version-3.3.0.md (hunk 6 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:6]: Fixed.

File: en/docs/release-notes/version-3.3.0.md (hunk 7 above)
Comment (praneesha) [reply en/docs/release-notes/version-3.3.0.md:42]: Changed the wording.

File: en/docs/release-notes/version-3.3.0.md (hunk 3 above)
Comment (anupama-pathirage) [reply en/docs/release-notes/version-3.3.0.md:35]: Agreed to some extension. That is a doubt I have too. It is not a text appearing in UI but a high-level feature in the tooling, right? Is it valid for such cases too?

File: en/docs/release-notes/version-3.3.0.md (hunk 4 above)
Comment (anupama-pathirage) [reply en/docs/release-notes/version-3.3.0.md:8]: Same as above

File: en/docs/release-notes/version-3.3.0.md (hunk 5 above)
Comment (anupama-pathirage) [reply en/docs/release-notes/version-3.3.0.md:5]: Same as above

**Pull request merged**